src/gui/modals/ControllerConnectionDialog.cpp

```diff
@@ -125,136 +124,122 @@ public slots:
+	for (Controller* c : Engine::getSong()->controllers())
 	{
-		m_userController->model()->addItem( c->name() );
+		m_userController->model()->addItem(c->name());
 	}
-	connect( m_userController->model(), SIGNAL(dataUnchanged()),
-			this, SLOT(userSelected()));
-	connect( m_userController->model(), SIGNAL(dataChanged()),
-			this, SLOT(userSelected()));
+	connect(m_userController->model(), SIGNAL(dataUnchanged()), this, SLOT(userSelected()));
+	connect(m_userController->model(), SIGNAL(dataChanged()), this, SLOT(userSelected()));
 
 
-	// Mapping functions
-	m_mappingBox = new TabWidget( tr( "MAPPING FUNCTION" ), this );
-	m_mappingBox->setGeometry( 8, 170, 240, 64 );
-	m_mappingFunction = new QLineEdit( m_mappingBox );
-	m_mappingFunction->setGeometry( 10, 20, 170, 16 );
-	m_mappingFunction->setText( "input" );
-	m_mappingFunction->setReadOnly( true );
+	// Mapping function
+	m_mappingBox = new TabWidget(tr("MAPPING FUNCTION"), this);
+	m_mappingBox->setGeometry(8, 170, 240, 64);
+	m_mappingFunction = new QLineEdit(m_mappingBox);
+	m_mappingFunction->setGeometry(10, 20, 170, 16);
+	m_mappingFunction->setText("input");
+	m_mappingFunction->setReadOnly(true);
 
 
-	// Buttons
+	// Buttons ("OK" and "Cancel" at the bottom)
 	auto buttons = new QWidget(this);
-	buttons->setGeometry( 8, 240, 240, 32 );
+	buttons->setGeometry(8, 240, 240, 32);
 
 	auto btn_layout = new QHBoxLayout(buttons);
-	btn_layout->setSpacing( 0 );
+	btn_layout->setSpacing(0);
 	btn_layout->setContentsMargins(0, 0, 0, 0);
 
 	auto select_btn = new QPushButton(embed::getIconPixmap("add"), tr("OK"), buttons);
-	connect( select_btn, SIGNAL(clicked()), 
-				this, SLOT(selectController()));
+	connect(select_btn, SIGNAL(clicked()), this, SLOT(selectController()));
 
 	auto cancel_btn = new QPushButton(embed::getIconPixmap("cancel"), tr("Cancel"), buttons);
-	connect( cancel_btn, SIGNAL(clicked()),
-				this, SLOT(reject()));
+	connect(cancel_btn, SIGNAL(clicked()), this, SLOT(reject()));
 
 	btn_layout->addStretch();
-	btn_layout->addSpacing( 10 );
-	btn_layout->addWidget( select_btn );
-	btn_layout->addSpacing( 10 );
-	btn_layout->addWidget( cancel_btn );
-	btn_layout->addSpacing( 10 );
+	btn_layout->addSpacing(10);
+	btn_layout->addWidget(select_btn);
+	btn_layout->addSpacing(10);
+	btn_layout->addWidget(cancel_btn);
+	btn_layout->addSpacing(10);
 
-	setFixedSize( 256, 280 );
+	setFixedSize(256, 280);
 
 	// Crazy MIDI View stuff
 	
-	// TODO, handle by making this a model for the Dialog "view"
-	ControllerConnection * cc = nullptr;
-	if( m_targetModel )
+	// TODO: handle by making this a model for the Dialog "view"
+	ControllerConnection* cc = nullptr;
+	if(m_targetModel)
 	{
 		cc = m_targetModel->controllerConnection();
-
-		if( cc && cc->getController()->type() != Controller::ControllerType::Dummy && Engine::getSong() )
+		if(cc && cc->getController()->type() != Controller::ControllerType::Dummy && Engine::getSong())
```

Comment:
```suggestion
		if (cc && cc->getController()->type() != Controller::ControllerType::Dummy && Engine::getSong())
```

"if" or "for" and "(" should have a space between.